modules/Bio/EnsEMBL/Variation/Pipeline/Remapping/PreRunChecks.pm

```diff
@@ -60,8 +60,8 @@ sub run {
 
   my $core_dbname = $cdba->dbc->dbname;
   $vdba->dbc()->sql_helper()->execute_update(-SQL => "DELETE FROM seq_region;", -PARAMS => [] );
-  my $sql = "INSERT INTO seq_region(seq_region_id, name)"
-    . "SELECT sr.seq_region_id, sr.name "
+  my $sql = "INSERT INTO seq_region(seq_region_id, name, coord_system_id) "
+    . "SELECT sr.seq_region_id, sr.name, sr.coord_system_id "
```

Comment:
Does the variation.coord_system table need to be updated?

Reply:
The data checks require that coord_system_id is also populated.

Reply:
If new coord_system_id are added to variation.seq_region.coord_system_id, then new records will need to be added to variation_coord_system table. It is the updates to this table that is I was querying.

Reply:
That is right. But this is done by the scrip ensembl-variation/scripts/misc/update_seq_region_coord.pl. I'm adding the coord_system_id here now because the coord_system_id cannot be NULL and doesn't have a default value. This didn't cause a problem before, but does now on codon.

Reply:
Thanks for the explanation.